DOC_2293-spark-0.2.0-requirements (#529)
tigergraph/server-docs: modules/data-loading/pages/load-from-spark-dataframe.adoc

include::partial$spark/pre_def_options.adoc[]

[NOTE]
.Authentication
====

> Authenticate with the username/password or secret to support token refresh, otherwise, make sure the lifetime of the token is long enough for the loading job.

Author: ```suggestion
Authenticate with the username/password or secret to automatically maintain a valid token; otherwise, make sure the lifetime of your token is long enough for the loading job.
```
